webapp/migrations/versions/292882794509_add_timezone_to_last_modified.py

```diff
@@ -17,7 +18,8 @@
 
 
 def upgrade():
-    op.alter_column('user', 'last_modified', type_=sa.TIMESTAMP(timezone=True))
+    if current_app.config['ENV'] in ('staging', 'production'):
+        op.alter_column('user', 'last_modified', type_=sa.TIMESTAMP(timezone=True))
```

Comment:
Couldn't we make this more explicit by checking for 
```suggestion
    if 'sqlite' in current_app.config['SQLALCHEMY_DATABASE_URI']:
        op.alter_column('user', 'last_modified', type_=sa.TIMESTAMP(timezone=True))
```
Or something else, I didn't check if we can access the config variables like that. The same would go for the `add_last_modified_column` migration.